server/service/src/sync/translations/message.rs

```diff
@@ -120,6 +123,9 @@ impl SyncTranslation for MessageTranslation {
         let created_date = created_datetime.date();
         let created_time = created_datetime.time();
 
+        // "unwrap_or" here would result in a string version of body json
+        let body = serde_json::from_str(&body).unwrap_or(serde_json::json!(body));
```

Comment:
I was really confused why to_value did not work on 'body', and tests were failing